### PR TITLE
Cut 0.1.61 release

### PR DESCRIPTION
**Pull request by mortid0**
- [x] Forked repo is synced with upstream -> github shows no code delta outside of the desired.
    https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
- [x] Tests are passing? https://github.com/microsoft/DeepGNN/blob/main/CONTRIBUTING.md#run-tests
- [x] Changelog and documentation updated.

Cut a new release after https://github.com/microsoft/DeepGNN/pull/233  merged

### CODE REVIEW

**Review by coledie — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.61] - 2022-10-04
```
Comment (coledie) [inline CHANGELOG.md:9]: Similarly update package_version in wheel action? https://github.com/microsoft/DeepGNN/blob/7b92623cdd31404b5996390fb8597c8efbbb056d/.github/actions/wheel/action.yml#L11

**Review by coledie — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mortid0) [reply CHANGELOG.md:9]: wheel action takes version as an argument. What exactly needs to be updated?

Comment (coledie) [reply CHANGELOG.md]: I see, but should the default at least be 9.9.99 or something so it is not confused with a real version?

Comment (mortid0) [reply CHANGELOG.md]: default is `0.1.1`. Which is smaller than any existing package, in case wheel is published accidentally.

**Pull request merged**